### PR TITLE
Decrease the size of the Dockerfile 

### PR DESCRIPTION
**Pull request by JasonPowr**

### CODE REVIEW

**Comment by JasonPowr** [comment]
While this pr isn't finished, the current build is a lot smaller, and the problem still exists with the build-source-image task

**Comment by JasonPowr** [comment]
This doesn't seem to fix the issue with the build-source-image task, but should probably be done at some point.

**Comment by JasonPowr** [comment]
As it stands the image builds and comes in at roughly 262MB, Im still running into an issue where the rekor server url is not being picked up and will investigate more tomorrow.

One note, I can see the correct url being picked up in the console.log, it just doesn't seem to be propagated to the actual ui

File: src/modules/components/SearchForm.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ import styles from "@patternfly/react-styles/css/components/Form/form";
 export interface FormProps {
 	defaultValues?: FormInputs;
 	isLoading: boolean;
-	onSubmit: (query: FormInputs) => void;
+	onSubmit: (_query: FormInputs) => void;
```
Comment (kahboom) [inline src/modules/components/SearchForm.tsx:24]: Apologies, as the lint warnings for this file and the following were driving me nuts so I made the fixes, but this change and the next are irrelevant to the PR

**Review by kahboom — APPROVED** [review]

**Pull request merged**